cluster/apps/default/echo-server/helm-release.yaml

```diff
@@ -27,6 +27,7 @@ spec:
       repository: docker.io/ealen/echo-server
       tag: 0.5.1
     ingress:
+      ingressClassName: "traefik"   
```

Comment:
Good find! Please move this under `enabled: true` and trim the whitespace at the end.

Reply:
Actually, this helm value is not exposed in the Helm Values
https://github.com/Ealenn/charts/blob/master/charts/echo-server/templates/ingress.yaml

Reply:
You can add this annotation instead `kubernetes.io/ingress.class: nginx` under `external-dns/is-public: "true"`